source/main.bmx

```diff
@@ -6106,6 +6113,9 @@ endrem
 		'remove from collection (reuse if possible)
 		GetNewsEventCollection().RemoveEndedNewsEvents()
 
+		If Not TSaveGame.autoSaveNow and TSaveGame.lastSaveTime > 0 and time - TSaveGame.lastSaveTime > TWorldTime.DAYLENGTH
```

Comment:
Hier _koennte_ man ueberlegen, irgendwo ein `GameConfig.AutoSaveInterval` einzufuehren - und das ist eben per default `TWorldTime.DAYLENGTH`  - koennte aber auch konfigurierbar sein (halber Tag, stuendlich, aller 2 Tage ...)

Reply:
Es war ja auch erstmal ein Proof of Concept. Im Ticket hatte ich ja auch unterschiedliche Varianten aufgelistet. Die einfachste Variante wäre ein Wert "autoSaveAfterGameHours" mit <=0 gar nicht und sonst "nach x Stunden im Spiel", d.h. 24 entspricht einmal täglich.
Die große Frage ist, wie (und wo) man den Wert sinnvoll für das aktuelle Spiel ändert. Vielleicht möchte man ja das Autosave an- und abschalten - in den Spieleinstellungen?

Reply:
Dazu hab ich keine Praeferenz.
Im Speichern-Dialog wuerde man es ja erstmalig sehen, wenn manuell gespeichert wuerde (bis dahin "unsichtbar", aka Default).
Eventuell also einfach "generisch" in den allgemeinen Spieleinstellungen machen ... also nicht spielstandsindividuell.

Reply:
Ich habe den Wert analog "RoomSlow" in die GameRules gepackt.
Nach dem Merge der Optimierungen würde ich das Menü analog des dort bearbeitbaren Prozentwerts erweitern.